### ISSUE TITLE
Orvis: Slideshow block in Portfolio Project shows as just a vertical stack of images

### ISSUE THREAD

**Issue by thesacredpath** [Automattic/themes#1294]
## Steps to replicate

1. Activate Orvis
2. Create portfolio project and add a slideshow block
3. Save and/or publish and preview project
4. If published, also look at project on the site

## Result

In editor, slideshow renders and shows as a slideshow. Preview shows images inserted into slideshow as vertical stack of the images, not as a slideshow. Confirmed on my test site with Orvis active. 

Slideshows inserted into pages or posts render correctly in preview and on site.

## Expected

Expected slideshow to appear in the portfolio project and render correctly

* User report: #12860098-hc
* Follow-up ticket: #2290204-zd

**Comment by kristarella** [comment]
I was not able to replicate this. The Slideshow block shows as a slideshow on my test site:
https://cld.wthms.co/2ZrT7j

**Comment by thesacredpath** [comment]
This is the project on my test site that shows the issue @kristarella: https://flippintestblog.wordpress.com/wp-admin/post.php?post=4171&action=edit. In editor, it shows as a slideshow, but when you preview, it shows as a list of the images.

**Comment by mdrockwell** [comment]
Another instance of this issue here: 15843810-hc
Follow-up Ticket: 2410633-zen

**Comment by jerrysarcastic** [comment]
Related to this issue, I helped a user who was trying to add the **Gallery Block** and it also rendered as a stack of images instead of as an image gallery. Steps to receate are the same as Richard noted for Slideshows, but insert a gallery block instead

In my tests using the classic editor, image galleries do function correctly so this seems to be a Gutenberg compatibility issue.

Video of the issue here:  https://d.pr/v/VwYiYN

Reported in #18070945-hc
Followup in #2652995-zen

**Comment by kriskarkoski** [comment]
Another case in 11412497-hc

**Comment by supernovia** [comment]
We have another case here:
https://wordpress.com/forums/topic/presentacion-dentro-de-portfolio/

**Comment by benchilcote** [comment]
Another case of this
Discovered here: #20940963-hc
Follow up here: #2957178-zen

**Comment by ktyfuller604** [comment]
Another report of this issue here (specifically the tiled gallery block on a project page): 
[22158387-hc](https://hud.happychat.io/chat/22158387)

**Comment by kristarella** [comment]
Yes, I just had this issue with another user 25133463-hc

The regular gallery block and the Tiled Gallery gets converted to single column.

In the Classic Block, the tiled gallery works as expected.

I'm going to change the title of the issue as it covers multiple gallery types

**Comment by fresatomica** [comment]
Another case in https://wordpress.com/forums/topic/orvis-galleries-dont-update-as-edited/.
Slideshow displaying as a stack of images.

**Comment by sophiegyo** [comment]
Another case in 3570948-zen
Slideshow block images show as a stack.

I was able to test and replicate on a Simple site as well, and also on a fresh jurassic ninja with newly installed Orvis.

**Comment by jwktje** [comment]
I really wish the orvis theme was in the repo somewhere so I could provide a PR to fix this

**Comment by synora** [comment]
@MaggieCabrera 👋 would you or someone from your team be able to look into this bug?
Another report in 3677016-zen

**Comment by MaggieCabrera** [comment]
I'll add this issue to our board, thanks for the reports

**Comment by jwktje** [comment]
@MaggieCabrera Thanks a bunch! Is there any way I could help provide the fix? Is the repo hosted somewhere where I can collaborate with a PR? I've studied the code and I'm 99% sure I know how to fix it. And a friend of mine has this issue in her website. I'd love to help provide a speedy fix

**Comment by jwktje** [comment]
Is there any update on this by any chance? Again, I'd love to help out and write a patch. My client is waiting on this fix

**Comment by Corinneba** [comment]
Hello I have the same problem! any updates on solutions?

**Comment by jwktje** [comment]
@Corinneba I have no update sadly. 2 months ago I first mentioned that I'd happily help fix this. I can understand if they have this theme in an internal repo that they don't wanna share. But the theme is years old and a bunch of other themes are in this repo already. It would be less work for them if they allowed the community to patch issues like this. Sorry for being a bit salty but it's frustrating when you have to tell your clients for weeks on end that your hands are tied.

**Comment by Corinneba** [comment]
@jwktje Couldn't agree more. It's very frustrating especially for people who are now choosing this template and who barely understand code, not knowing that it doesn't work with Gutenberg. Wordpress shouldn't recommend it anymore!! I spent hours to fix small things here and there.

**Comment by ktyfuller604** [comment]
Another report here: 3891744-zen

**Comment by Jaykame** [comment]
Reporting similar issue here: 17604936-hc

**Comment by zdenys** [comment]
4900881-zen

**Comment by Robertght** [comment]
Orvis is being used by at least 196,945 websites. cc @jeffikus 

I can replicate this consistently on Simple and Atomic sites.

**Comment by edwinho89** [comment]
5597194-zd-woothemes

In conjunction to the above, user also experienced spacer block being ignored on https://github.com/Automattic/themes/issues/2262

**Comment by rakshitwesley** [comment]
Another user reported the issue here: 37046127-hc

Ticket: 5721682-zd-woothemes

**Comment by aleone89** [comment]
Another report of this here: https://wordpress.com/forums/topic/slideshow-widgets-2/?view=all

**Comment by Robertght** [comment]
Another reported in 9336738-zen

**Comment by Robertght** [comment]
@dsas Would this be something the team can look into?

Also, should issues as old as this be considered for retriaging?

**Comment by Robertght** [comment]
PS: It looks like the Gallery and the Tiled Gallery block have the same fate - they display as a single column too.

**Comment by kudama** [comment]
Has anyone had luck fixing this? Is there a code snippet we can use on the single portfolio page template to fix?
